client/components/login/LoginForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-sheet tile elevation="0" color="white">
+    <v-container class="py-16">
+      <v-row>
+        <v-col cols="6" class="mx-auto">
+          <h2 style="text-align: center; margin-bottom: 1rem">Login</h2>
+          <v-form ref="form" v-model="valid" lazy-validation>
```

Comment:
Ideally, we would want the form to handle submission rather than on click of a button.

```suggestion
          <v-form ref="form" v-model="valid" @submit.prevent.stop="validate()">
```